scene/gui/color_picker.cpp

```diff
@@ -399,7 +399,7 @@ void ColorPicker::_screen_input(const InputEvent &ev)
 		Image img =r->get_screen_capture();
 		if (!img.empty())
 			last_capture=img;
-			r->queue_screen_capture();
```

Comment:
It queues an screen capture to get the color the next time the cursor is moved, if there's an animation runing or an animated shader this is needed to update properly. This should be in fact happening in SceneTree's idle frame to capture every color, but i decided to wait until find it's really needed.
In short, put that line back :)